ingest/cell_metadata.py

```diff
@@ -103,13 +162,13 @@ def validate_type_annotations(self):
             valid = True
         return valid
 
-    def validate_against_header_count(self, list):
+    def validate_against_header_count(self):
         """Metadata header and type counts should match.
 
         :return: boolean   True if valid, False otherwise
         """
         valid = False
-        if not len(self.headers) == len(list):
+        if not len(self.headers) == len(self.metadata_types):
```

Comment:
thanks for catching that I did not refactor properly when I added metadata_types to the CellMetadata object!